### PR TITLE
Simplify public API

### PR DESCRIPTION
**Pull request by anaschwendler**
This PR is related to #59.

So here, I refactor the names of the modules to simple ones.
In that journey, I noticed that we've completely forgot to check some files while working on refactoring the toolbox to accept both Chamber of Deputies and Federal Senate.

image

As we see in the image, it still keeps the format from before Federal Senate introduction  👀 
But I don't know if we just wait for the #87 to fix it, or something.

Help will be accepted <3

### CODE REVIEW

**Comment by anaschwendler** [comment]
Ok, I think that I find a way to update the sphinx documentation.
What I've done was:
`make rst` into the `/docs` folder

So, I am not sure if it was the right way, I'll be waiting for reviews :)

File: docs/source/serenata_toolbox.rst [hunk 1]
```diff
@@ -1,41 +1,14 @@
 serenata_toolbox package
 ========================
 
-Submodules
-----------
+Subpackages
```
Comment (cuducos) [inline docs/source/serenata_toolbox.rst]: AFAIK they are submodules, not subpackages… but I assume this is merely 💅 So the real question is: have you changed it yourself or was it changed automagically? If it was automagically just let it be ; )

Comment (anaschwendler) [inline docs/source/serenata_toolbox.rst]: Changed automagically! 
So, I can change it manually, but it was generated, so I don't know.

Comment (anaschwendler) [inline docs/source/serenata_toolbox.rst]: I just run the makefile and 🎉

Comment (cuducos) [inline docs/source/serenata_toolbox.rst]: In `sphinx` we trust then.

**Comment by anaschwendler** [comment]
Ok, I guess that now everything is right!

**Comment by cuducos** [comment]
> Ok, I guess that now everything is right!

Running integration tests and merge it as soon as it finishes ; )

**Pull request merged**